Essentials/src/main/java/com/earth2me/essentials/utils/LocationUtil.java

```diff
@@ -274,7 +275,8 @@ public static boolean shouldFly(IEssentials ess, final Location loc) {
             }
         }
 
-        return y < 0;
+        //If not then check if player is not in void
```

Comment:
Don't you mean the opposite here? It looks like this is checking if the player *is* in the void.

Reply:
Yes my bad, I should have think twice about it before pushing it, I'll change that